docs/source/sci_api_tutorial/mutate_stru.rst

```diff
@@ -0,0 +1,189 @@
+
+    - *Avaible strageties:*
+        Substitution:
+
+        Direct replacement of the side-chain:
+
+        - tleap_min (https://pubs.acs.org/doi/full/10.1021/acs.jcim.1c01424)
+            The most simple way to get a mutant structure. It 1. place the new side-chain using a constant conformation (relative to backbone) and 2. relax the crude mutant structure using MM minimization.
+
+            Consider limited change of other side chains in MM minimization
+            Not consider backbone change
+
+            * This method particularly has problem when mutating a small residue to a larger one. In this case, collision may appears in the mutated structure and the MM minimization is responsible for resolving it. But in extreme cases, there are unresolvable collision such as the carbon chain is trapped in a phenyl ring. And currently we don't have a method to detect such trapping. The method is only used as a place holder for 1st version EnzyHTP. We have encounter any problem brought by the accuracy of the mutation when using this method in workflows but users should be cautious with it and consider it a potential source of absured results.
+        
+        Side-chain rotamer library:
+        (most used in the field)
+        
+        - SCWRL4 (http://dunbrack.fccc.edu/lab/scwrl)
+            
+            | Not consider other side-chain change
+            | Not consider backbone change
+
+        - PyMol (https://github.com/schrodinger/pymol-open-source)
+            
+            | Not consider other side-chain change
+            | Not consider backbone change
+            
+        - Phyre2
+            
+            | Consider other side-chain change
+            | Not consider backbone change
+
+            * seems having a derived pipeline Missense3D addressing the challenge (https://www.sciencedirect.com/science/article/pii/S0022283619302037?via%3Dihub#s0050)
+
+            * related discussion in its website (http://www.sbg.bio.ic.ac.uk/phyre2/html/help.cgi?id=help/faq)
+
+        Machine learning methods:
+
+        - Packpred (http://cospi.iiserpune.ac.in/packpred/, https://www.frontiersin.org/articles/10.3389/fmolb.2021.646288/full, https://github.com/kuanpern/PackPred)
+
+          * find a summary of the missence mutation in the intro of the paper
+            Unknown
+
+        MCMC search globally in side-chains:
+
+        - Modeller
+
+            | Fully consider other side-chain change
+            | Not consider backbone change
+
+        - SWISSMODEL
+  
+            | Fully consider other side-chain change
+            | Not consider backbone change
+
+
+    - *Insertion/Deletion:*
+
+        - Phyre4
+  
+            see http://www.sbg.bio.ic.ac.uk/phyre2/html/help.cgi?id=help/faq
+            works mainly <5 AA change
+
+
+Example Code
+==============================================
+
+Mutate a target protein
+---------------------------------------------------------
+
```

Comment:
Please provide more guide in this example and make them seperate examples. (see QZ's branch)